pulp_rpm/app/tasks/publishing.py

```diff
@@ -342,7 +342,15 @@ def publish(
     )
     with tempfile.TemporaryDirectory("."):
         with RpmPublication.create(repository_version) as publication:
-            publication.metadata_checksum_type = get_checksum_type("primary", checksum_types)
+            kwargs = {}
+            first_package = repository_version.content.filter(
+                pulp_type=Package.get_pulp_type()
+            ).first()
+            if first_package:
+                kwargs["default"] = first_package.cast().checksum_type
+            publication.metadata_checksum_type = get_checksum_type(
+                "primary", checksum_types, **kwargs
+            )
```

Comment:
Could we file an issue for removing this and link to it here?  And on the issue, link to this PR or commit, so that we can just "git revert ...." when the time comes.